docs/narr/viewconfig.rst

```diff
@@ -494,6 +494,16 @@ configured view.
 
   .. versionadded:: 1.4a3
 
+``is_authenticated``
+
+  This value, if specified, should be either ``True`` or ``False``.  If it is
```

Comment:
```suggestion
  This value, if specified, must be either ``True`` or ``False``.  If it is
```

This might need clarification pending outcome of comment https://github.com/Pylons/pyramid/pull/3598/files#diff-dcbd22655007a5407eaee38bdc2e1cf5R289

Reply:
`should` is taken from the documentation of other predicates.

Reply:
Given that, should we document what happens if the value is neither `True` nor `False`, or leave it open for the developer to find out?

Reply:
I have changed to `must`.  Is more documentation needed?

Reply:
`must` is more definitive, so I think this is good to go.